### ISSUE TITLE
Publish for Scala Native 0.5.x

### ISSUE THREAD

**Issue by Gedochao** [typelevel/toolkit#171]
It'd be cool if the toolkit could work with Scala Native 0.5.x (0.5.1 as of me writing this)

```bash
scala-cli -e 'println("Hello")' --toolkit typelevel:0.1.25 --native --native-version 0.5.1
# Downloading compiler plugin org.scala-native:::nscplugin:0.5.1
# Downloading Scala Native CLI
# Downloading 4 dependencies and 2 internal dependencies
# [error]  Error downloading org.typelevel:toolkit_native0.5_3:0.1.25
# [error]   not found: /Users/pchabelski/.ivy2/local/org.typelevel/toolkit_native0.5_3/0.1.25/ivys/ivy.xml
# [error]   not found: https://repo1.maven.org/maven2/org/typelevel/toolkit_native0.5_3/0.1.25/toolkit_native0.5_3-0.1.25.pom
# [error]   not found: /Users/pchabelski/Library/Caches/ScalaCli/local-repo/1.2.2/org.typelevel/toolkit_native0.5_3/0.1.25/ivys/ivy.xml
# [error]   No fallback URL found
# [error] COMMAND_LINE
```

**Comment by TonioGela** [comment]
Hey @Gedochao, I totally agree with you! At the moment we're tracking the status of the migration towards Scala Native 0.5.x for the whole ecosystem here: https://github.com/orgs/typelevel/projects/2

We will soon publish everything up until Cats really soon, but then we'll have to decide what to do with Cats Effect, if releasing a native version that still works single-threadedly or waiting for intelligent people like @djspiewak to craft something multi-threaded.

Given that the toolkit depends on a lot of stuff it's the last piece that will get published for sn 0.5, as we'll have to wait for things like http4s.

Thanks for being so interested :)

**Comment by armanbilge** [comment]
@Gedochao after 2 years 😅 we have all the releases lined up to do this! We just need a scala-cli release with https://github.com/VirtusLab/scala-cli/pull/4229 😅

**Comment by Gedochao** [comment]
Nice! I mean to release Scala CLI this week, so stay tuned.

**Comment by Gedochao** [comment]
https://github.com/VirtusLab/scala-cli/releases/tag/v1.13.0

**Comment by Hombre-x** [comment]
Its ok to close this issue now? Referencing #315

**Comment by armanbilge** [comment]
Yes! Landed in [v0.1.30](https://github.com/typelevel/toolkit/releases/tag/v0.1.30) and [v0.2.0](https://github.com/typelevel/toolkit/releases/tag/v0.2.0). :)
